internal/database/sub_repo_perms_store.go

```diff
@@ -269,3 +509,9 @@ DELETE FROM sub_repo_permissions WHERE user_id = %d
 `, userID)
 	return s.Exec(ctx, q)
 }
+
+// IPsNotSyncedError is returned when the IP addresses associated with a sub-repository-permissions entry have not been synced yet.
+//
+// Currently, this means that the permissions provider for the given service has not been updated to include the IP addresses with each
+// entry it inserts.
+var IPsNotSyncedError = errors.New("The IP addresses associated with this sub-repository-permissions entry have not been synced yet.")
```

Comment:
Go style guide: No punctuation and capitalization in error messages

```suggestion
var IPsNotSyncedError = errors.New("the IP addresses associated with this sub-repository-permissions entry have not been synced yet")
```